demos/swift/js/somepackage/RequiredClass.js

```diff
@@ -0,0 +1,30 @@
+/**
```

Comment:
This is the wrong license header. Same for other files.

Reply:
Updated here and a bunch of other files.